plugins/nlp/nlp.hpp

```diff
@@ -35,4 +35,15 @@
 #include "nlp_eng.hpp"
 #include <sstream>
 
+extern "C"
+{
+
+#ifdef ECL_NLP_EXPORTS
+    ECL_NLP_API bool getECLPluginDefinition(ECLPluginDefinitionBlock *pb);
+    ECL_NLP_API void setPluginContext(IPluginContext * _ctx);
 #endif
+
+ECL_NLP_API void ECL_NLP_CALL AnalyzeText(size32_t & tgtLen, char * & tgt, size32_t anaLen, const char * ana, size32_t txtLen, const char * txt);
```

Comment:
I am not sure that this should be inside the extern "C" given the definition in the ecllib file